src/app/shared/services/app.service.ts

```diff
@@ -101,6 +101,20 @@ export class NSSService {
     public get showManageCitationsModal(): any {
         return this._showHideManageCitationsModal.asObservable();
     }
+    // Toggle Add/Delete Citation in managecitations modal
+    private addCitations = new BehaviorSubject<boolean>(true);
+    currentAddCitations = this.addCitations.asObservable();
+
+    showAddCitations(bool: boolean){
+        this.addCitations.next(bool);
+    }
```

Comment:
This all works well, but could probably be combined with the showManageCitations modal stuff, similarly to how we're sending things along to the showHideAddRegressionRegion modal, so you're only setting one subject when you want the modal shown.